app/templates/about/legal.hbs

```diff
@@ -1,41 +1,39 @@
+    </p>
+    <p>
+      We often grant licenses for use of the Ember logo for non-commercial community use cases. If you think you might be one of them, let's talk. We always aim to be reasonable.
+    </p>
+    <p>
+      The Tomster in particular has many more restrictions and should not be used without express authorization. {{link-to "You can view the various official Tomsters" "mascots"}}, and check out {{link-to "our growing FAQ" "mascots.faq"}}.
+    </p>
 
-  <h2>Non-Legalese TL;DR</h2>
+    <h2>The Ember License</h2>
+    <p>
+      The most current Ember license details <a href="https://github.com/emberjs/ember.js/blob/master/LICENSE">can be found on GitHub</a>.
+    </p>
 
-  <p>
-    Ember's strong brand and identity are an important part of the project. We're protective of it! Please be respectful and <a href="mailto:leah@emberjs.com">reach out</a> if you're unsure if your use is acceptable. Visit the {{link-to "branding page" "logos"}} to learn more about what is and isn't permitted, and to download logos.
-  </p>
-  <p>
-    We often grant licenses for use of the Ember logo for non-commercial community use cases. If you think you might be one of them, let's talk. We always aim to be reasonable.
-  </p>
-  <p>
-    The Tomster in particular has many more restrictions and should not be used without express authorization. {{link-to "You can view the various official Tomsters" "mascots"}}, and check out {{link-to "our growing FAQ" "mascots.faq"}}.
-  </p>
-
-  <h2>The Ember License</h2>
-
-  <p>
-    The most current Ember license details <a href="https://github.com/emberjs/ember.js/blob/master/LICENSE">can be found on GitHub</a>.
-  </p>
-
-  <h2>Documentation Images</h2>
-
-  <p>
-    The graphs, diagrams and charts in the <a href="/guides">Ember Guides</a> have been released under the Creative Commons Attribution-NonCommercial-ShareAlike (CC BY-NC-SA) license.
-  </p>
-  <img alt="CC, By, Non Commercial, Share Alike" class="pull-right" src="/images/legal/license-cc_by_nc_sa.png">
-  <p>
-    This license allows others to remix, tweak, and build upon the instructional materials non-commercially, as long as they credit Ember and license any new creations under the identical terms.
-  </p>
-  <p>
-    This license does <strong>not</strong> include any uses of the Tomster included in the Guides. Please reach out for clarification on any ambiguous images before using.
-  </p>
-</section>
+    <h2>Documentation Images</h2>
+    <p>
+      The graphs, diagrams and charts in the <a href="/guides">Ember Guides</a> have been released under the Creative Commons Attribution-NonCommercial-ShareAlike (CC BY-NC-SA) license.
+    </p>
+    <img alt="CC, By, Non Commercial, Share Alike" class="pull-right" src="/images/legal/license-cc_by_nc_sa.png">
```

Comment:
this should be `alt="" role="presentation"` since the preceding paragraph describes the license.

Reply:
👍 I also worked it out to wrap correctly now.